src/custom/state/claim/hooks/index.ts

```diff
@@ -279,6 +283,36 @@ export function useUserClaims(account: Account, optionalChainId?: SupportedChain
   return { claims: claimKey ? claimInfo[claimKey] : null, isLoading }
 }
 
+let fetch_deployment_timestamp_promise: Promise<number> | null = null
+function fetchDeploymentTimestamp(vCowContract: VCowType) {
+  if (!fetch_deployment_timestamp_promise) {
+    fetch_deployment_timestamp_promise = vCowContract.deploymentTimestamp().then((ts) => {
+      console.log(`Deployment timestamp in seconds: ${ts.toString()}`)
+      return ts.mul('1000').toNumber()
+    })
+  }
+
+  return fetch_deployment_timestamp_promise
+}
+
+/**
+ * Creates a ref that can be used to solve the issue of
+ * "Can't perform a React state update on an unmounted component."
+ */
+function useIsMounted() {
```

Comment:
why not do this in a more general place and export it. Its a general solution, right?